test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,73 @@
+const axios = require('axios');
+const { expect, assert } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('Consume POST and PATCH Method', () => {
+  const user = 'MariaJose319';
+  const repo = 'BootcampQA_PetStoreChallenge';
+  const urlBase = 'https://api.github.com';
+  let userResponse;
+  let reposResponse;
+  let createIssueResponse;
+  let patchResponse;
+
+  before(async () => {
+    userResponse = await axios.get(`${urlBase}/user`, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+  });
+
+  it('verify that the user has at least a public repository', async () => {
+    expect(userResponse.status).to.equal(StatusCodes.OK);
+    expect(userResponse.data.public_repos).to.satisfy((num) => num > 0);
```

Comment:
```suggestion
    expect(userResponse.data.public_repos).to.be.above(0);
```
Is an assert more suited for this data type (size of a list)